wss/1.0/modules/ROOT/partials/transformation-note.adoc

```diff
@@ -0,0 +1,4 @@
+[NOTE]
+--
+Transform operations don't take into account the cryptographic signatures and can end up modifying them.
```

Comment:
What do you mean by "take into account"?

Reply:
Ignores them as any other XML values. It will not try to preserve formatting.